greedy_approach/JobSequencing.c

```diff
@@ -0,0 +1,79 @@
+#include<stdio.h>
```

Comment:
Please describe how the algorithm works, a Wikipedia or any other source for algorithm explanation, what the algorithm does, etc..